scrapy/logformatter.py

```diff
@@ -125,7 +125,7 @@ def dropped(
         # that is used in DropItem exception, so a default value is provided
         # inside the first getattr() below
         log_level = getattr(exception, "log_level", "WARNING")
-        severity = getattr(logging, log_level.upper())
+        severity = getattr(logging, log_level)
```

Comment:
```suggestion
        if isinstance(severity, str):
            severity = getattr(logging, log_level)
```

In case they pass `logging.INFO` or similar.